Index: icecream/client/Makefile.am
===================================================================
--- icecream/client/Makefile.am	(revision 556877)
+++ icecream/client/Makefile.am	(working copy)
@@ -3,7 +3,7 @@
 bin_PROGRAMS = icecc
 pkglib_SCRIPTS = icecc-create-env
 icecc_SOURCES = main.cpp arg.cpp cpp.cpp local.cpp remote.cpp util.cpp md5.c safeguard.cpp
-icecc_LDADD = ../services/libicecream.la
+icecc_LDADD = ../services/libicecc.la
 
 CLEANFILES = icecc++
 
Index: icecream/daemon/Makefile.am
===================================================================
--- icecream/daemon/Makefile.am	(revision 556877)
+++ icecream/daemon/Makefile.am	(working copy)
@@ -3,4 +3,4 @@
 
 sbin_PROGRAMS = iceccd
 iceccd_SOURCES = ncpus.c main.cpp serve.cpp workit.cpp environment.cpp load.cpp
-iceccd_LDADD = ../services/libicecream.la
+iceccd_LDADD = ../services/libicecc.la
Index: icecream/services/Makefile.am
===================================================================
--- icecream/services/Makefile.am	(revision 556877)
+++ icecream/services/Makefile.am	(working copy)
@@ -1,17 +1,17 @@
 INCLUDES = -I$(srcdir)/../minilzo
 
-lib_LTLIBRARIES = libicecream.la
+lib_LTLIBRARIES = libicecc.la
 
-libicecream_la_SOURCES = job.cpp comm.cpp getifaddrs.cpp logging.cpp tempfile.c
-libicecream_la_LIBADD = ../minilzo/libminilzo.la
-libicecream_la_CXXFLAGS = -fPIC -DPIC
+libicecc_la_SOURCES = job.cpp comm.cpp getifaddrs.cpp logging.cpp tempfile.c
+libicecc_la_LIBADD = ../minilzo/libminilzo.la
+libicecc_la_CXXFLAGS = -fPIC -DPIC
 
 ice_HEADERS = job.h comm.h
-icedir = $(includedir)/icecream
+icedir = $(includedir)/icecc
 
 sbin_PROGRAMS = scheduler
 scheduler_SOURCES = scheduler.cpp
-scheduler_LDADD = libicecream.la
+scheduler_LDADD = libicecc.la
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = icecream.pc
