Index: icecream/services/icecream.pc.in
===================================================================
--- icecream/services/icecream.pc.in	(revision 556877)
+++ icecream/services/icecream.pc.in	(working copy)
@@ -1,14 +1,14 @@
-# icecream pkg-config file
+# icecc pkg-config file
 
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
 
-Name: icecream
-Description: icecream is a library for connecting to icecream schedulers
+Name: icecc
+Description: icecc is a library for connecting to icecc schedulers
 Version: @VERSION@
 Requires:
 Conflicts:
-Libs: -L${libdir} -licecream
+Libs: -L${libdir} -licecc
 Cflags: -I${includedir}
